### ISSUE TITLE
Try out `nbinteract`

### ISSUE THREAD

**Issue by mandli** [clawpack/riemann_book#123]
I just became aware of the [`nbinteract`](https://www.nbinteract.com/) project by @SamLau95 that can turn notebooks into interactive webpages.  Might be an interesting way to put this book up in more places.

**Comment by rjleveque** [comment]
Looks pretty cool, thanks!

**Comment by ketch** [comment]
Yes, that is just the kind of thing I'd love to have.  However, it seems to be limited (at least for now) to very specific, simple things (e.g. one plot with one line) and wouldn't accommodate most of what is in the book, as far as I can tell.  I've asked the developers whether they plan to make more complex plots possible.

**Comment by ketch** [comment]
I take back my last comment -- the dev says that nbinteract should be able to publish any notebook as a webpage with the interacts intact.  He even seems to be using our notebooks as a test case:

https://github.com/SamLau95/nbinteract/issues/45

However, I haven't gotten nbinteract.publish() to work at all yet on my machine.

**Comment by SamLau95** [comment]
Hi @ketch , we've taken your repo's notebooks and are in the process of converting them into online webpages. We've actually learned a lot from this process and will release a new version of nbinteract soon that should make conversion a lot easier on your end.

Expect a comment on this thread later this week!

**Comment by ketch** [comment]
Thanks @SamLau95; we're excited.  Having functional widgets in the HTML version of our book would be a huge improvement.
